src/internal/hooks/theme/useMessageReceivers.ts

```diff
@@ -32,6 +33,7 @@ export const useThemeMessageReceivers = () => {
         ? jsonFromEscapedJsonString(payload?.history)
         : [],
       index: payload?.index ?? 0,
+      hash: payload?.hash ?? uuidv4(),
```

Comment:
uuid should already be set (going forward). We can clear the localStorage and themeSaveState tables so we don't need this. You can also setThemeSaveState directly from the payload instead of recreating the object here. The one thing I'm noticing though is that we json escape it here but we don't in the getSaveState receiver. We should make them consistent. Idk which is better to use. json escape right here or do it later? Probably here, which means we probably want to update getSaveState. We can do that in another PR though.